cms/static/js/views/course_info_helper.js

```diff
@@ -2,6 +2,7 @@ define(['codemirror', 'js/utils/handle_iframe_binding', 'utility'],
     function(CodeMirror, IframeBinding) {
         var editWithCodeMirror = function(model, contentName, baseAssetUrl, textArea) {
             var content = rewriteStaticLinks(model.get(contentName), baseAssetUrl, '/static/');
```

Comment:
Rather than exposing these two replace methods can we extract them from here into a generic method which can either decide which one to call or whose only responsibility would be to rewritestaticurls? So will make the code more readable.